maps/nerva/datums/nerva_jobs.dm

```diff
@@ -81,7 +81,7 @@
 	return "You are the First Officer, and second in command of the ICS Nerva. As the clear second in command of the ship, your job is to work with the captain to run the ship, and take charge of navigation according to the captain's orders. Moreover, if there is no second officer, your job is also to oversee personnel issues and organize away missions. In the event of combat, your job is to work with the Chief of Security to coordinate the ship's defence."
 
 /datum/job/captain
-	supervisors = "yourself, as you are the owner of this ship and the sole arbiter of its destiny. However, be careful not to anger NanoTrasen and the other factions that have set up outposts in this sector, or your own staff for that matter. It could lead to your undoing"
+	supervisors = "yourself and your counterpart, as you are the owner of this ship and the sole arbiter of its destiny, alongside your counterpart.. However, be careful not to anger NanoTrasen and the other factions that have set up outposts in this sector, or your own staff for that matter. It could lead to your undoing"
```

Comment:
Changing it to 'are the sole arbiters' would read better IMO; it's an awkward reduplication and so it would read better if it was changed.